library/src/CMakeLists.txt

```diff
@@ -799,10 +799,15 @@ if( BUILD_WITH_TENSILE )
   endif()
   # For ASAN package, Tensile library files(which are not shared libraries) are not required
   if( NOT ENABLE_ASAN_PACKAGING )
+    if( BUILD_SHARED_LIBS )
+      set( TENSILE_DATA_COMPONENT_NAME ${CMAKE_INSTALL_DEFAULT_COMPONENT_NAME} )
+    else()
+      set( TENSILE_DATA_COMPONENT_NAME clients-common )
+    endif()
```

Comment:
So I am not sure of this convention.  Clients are users of the library.   The library runtime depends on TensileData.  Downstream consumers of the library don't depend on clients-common ?

Reply:
@evetsso and @lawruble13 please chime in.

Reply:
This had been my initial recommendation, but on discussion with you folks I now agree that this isn't the way to approach it. We need a separate package to contain the Tensile files in a static context.

Reply:
Does this component name get turned into a package name when we start making packages?